src/view.rs

```diff
@@ -163,6 +183,7 @@ pub trait View {
         layout_rect
     }
 
+    // QUESTION: what's the difference between this and `layout`?
```

Comment:
@jrmoulton do you know the difference between compute_layout and layout?

Reply:
I'm not 100% certain actually but the way I've been using them is that the layout is used to update taffy and update the layout of the taffy node. Then the compute_layout gets the taffy layout and then gets the rectangle that is the outer size of the view

Reply:
I think this is kinda answered now based on what I discerned from @dzhou121's explanation of the render lifecycle.